services/api-server/tests/unit/test_services_rabbitmq.py

```diff
@@ -56,6 +56,7 @@ def app_environment(
     mocker: MockerFixture,
 ) -> EnvVarsDict:
     # do not init other services
+    mocker.stopall()
```

Comment:
this is a bit suspicious. why do we need this?

Reply:
Yes, I didn't see how else to do this. The point is that I only need the rabbitmq client for very few tests, so in the `app_environment` fixture in `conftest.py` I mock the fcn which sets up the rabbitmq client [here](https://github.com/ITISFoundation/osparc-simcore/blob/38e1f63203389f3867a590fa81baf4f229ebc03d/services/api-server/tests/unit/conftest.py#L65). And apparently when I override fixtures, the toplevel fixture is called first. Hence, when I arrive here the rabbitmq client is mocked. Hence, I need to remove that mock again. Do you see a better way to do this?

Reply:
Sorry for the noise, I fixed this.